### DISCUSSION TITLE
Feature request: community MDI

### DISCUSSION THREAD

**Discussion by nutteloost** [home-assistant/frontend#11353] [Configuration] [answered]
I'm missing some community Material Design Icons in Home Assistant.

I would like to use mdi:curtains, mdi:curtains-closed, mdi:soundbar but they are not visible in Home Assistant version 2021.9.3.

**Comment by frenck** [comment, marked as answer]
Update Home Assistant :)

**Reply by nutteloost** [reply]
Having some problems with the Tuya integration (doesn't support my curtains) have to wait for 2022.02 :( 
So the icons are in the updated version of HA, great, thanks!

**Reply by spacegaier** [reply]
Yes, whenever a new MDI version is released we update it in our frontend and then ship them with the next release that contains a frontend update. Currently we are on v6.5.95 (https://github.com/home-assistant/frontend/pull/10618)
